src/main/java/me/karakelley/http/Request.java

```diff
@@ -82,10 +82,11 @@ private void setHeaders() {
 
   private void setRequestLine() {
     String requestLine;
-    String[] parsedLine;
+    String[] parsedLine = new String[2];
```

Comment:
Because of the first `if` block where it checks for null, since we may not hit that line and initialize `parsedLine` then it may not be initialized therefore it's initialized on line 85, outside of the `if` block.